src/utils.ts

```diff
@@ -1,20 +1,25 @@
+import { isNumber, isString } from 'lodash';
 import Decimal from 'decimal.js';
 
 import { CENT_DECIMAL } from './constants';
 
-export function insertIf(condition: boolean, element: any): any[] {
+export function insertIf<T>(condition: boolean, element: T): T[] {
   return condition ? [element] : [];
 }
 
-export function getPercentValue(value?: null | string): string {
-  if (typeof value === 'undefined' || value === null || value === '') {
+function _isDecimalValue(value: unknown): value is Decimal.Value {
+  return (isString(value) && value !== '') || isNumber(value) || Decimal.isDecimal(value);
```

Comment:
A little confused here, how does checking if it is a string and not empty suffice for it being a `Decimal.Value`?

Reply:
You can see here the definition of Decimal.Value is string or number or `Decimal`

![image](https://user-images.githubusercontent.com/796717/134061944-6464b27e-c70d-42c6-82d1-ffdd746eddae.png)